hedera-node/configuration/mainnet/bootstrap.properties

```diff
@@ -6,4 +6,3 @@ contracts.sidecars=
 hedera.recordStream.enableTraceabilityMigration=false
 autoCreation.enabled=true
 lazyCreation.enabled=false
```

Comment:
Shouldn't we remove `lazyCreation.enabled` as well ?

Reply:
`lazyCreation.enabled` _is_ in `0.33`, [afaict](https://github.com/hashgraph/hedera-services/blob/v0.33.2/hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties#L51).

Reply:
I was wondering because of the stack trace
```
java.lang.IllegalStateException: 'bootstrap.properties' contains unrecognized properties: [cryptoCreateWithAliasAndEvmAddress.enabled, lazyCreation.enabled]!
```

Reply:
You could get that on a `0.33` tag?

Reply:
That was the stack trace devops posted. Did not try on local